cardano-cli/src/Cardano/CLI/Byron/Parsers.hs

```diff
@@ -758,4 +758,3 @@ parseSigningKeyFile opt desc = SigningKeyFile <$> parseFilePath opt desc
 parseGenesisFile :: String -> Parser GenesisFile
 parseGenesisFile opt =
   GenesisFile <$> parseFilePath opt "Genesis JSON file."
```

Comment:
Please undo the setting in your editor that automatically removes newlines from files. It's unnecessary diff.